monitoring/opencensus/main.py

```diff
@@ -0,0 +1,120 @@
+"""
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+import random
+import time
+
+from flask import Flask
+from opencensus.ext.prometheus import stats_exporter as prometheus
+from opencensus.stats import aggregation as aggregation_module
+from opencensus.stats import measure as measure_module
+from opencensus.stats import stats as stats_module
+from opencensus.stats import view as view_module
+from opencensus.tags import tag_map as tag_map_module
+
+from prometheus_flask_exporter import PrometheusMetrics
+
+# set up measures
+m_request_count = measure_module.MeasureInt(
+    "python_request_count", "total requests", "requests"
+)
+m_failed_request_count = measure_module.MeasureInt(
+    "python_failed_request_count", "failed requests", "requests"
+)
+m_response_latency = measure_module.MeasureFloat(
+    "python_response_latency", "response latency", "s"
+)
+
+# set up stats recorder
+stats_recorder = stats_module.stats.stats_recorder
+
+# set up views
+latency_view = view_module.View(
+    "python_response_latency",
+    "The distribution of the latencies",
+    [],
+    m_response_latency,
+    aggregation_module.DistributionAggregation(
+        [0, 1000, 2000, 3000, 4000, 5000, 6000, 7000, 8000, 9000, 10000]
+    ),
+)
+
+request_count_view = view_module.View(
+    "python_request_count",
+    "total requests",
+    [],
+    m_request_count,
+    aggregation_module.CountAggregation(),
+)
+
+failed_request_count_vew = view_module.View(
+    "python_failed_request_count",
+    "failed requests",
+    [],
+    m_failed_request_count,
+    aggregation_module.CountAggregation(),
+)
+
+
+# register views
+def registerAllViews(view_manager: stats_module.stats.view_manager) -> None:
```

Comment:
Please use `camel_case` for the function names. https://google.github.io/styleguide/pyguide.html#316-naming

Reply:
Fixed, thank you!